finetune/adapter.py

```diff
@@ -131,10 +134,8 @@ def main(fabric: L.Fabric, devices: int, seed: int, config: Config, io: IOArgs,
     # strict=False because missing keys due to Adapter weights not contained in state dict
     load_checkpoint(fabric, model, checkpoint_path, strict=False)
 
-    fabric.seed_everything(1337 + fabric.global_rank)
```

Comment:
You removed this because the dataloaders seed themselves, right? But isn't the new version missing the `global_rank` increase?

Reply:
The seeding is for the dataset/dataloader yes and is now handled in the module.
The global rank thing is not necessary anymore, because `fabric.setup_dataloaders()` will configure the distributed sampler in the DataLoader.